vhdl_lang/src/syntax/design_unit.rs

```diff
@@ -559,7 +579,8 @@ end architecture arch_name;
             [simple_architecture(
                 WithDecl::new(code.s1("arch_name").ident()),
                 code.s1("myent").ident(),
-                Some(code.s("arch_name", 2).pos())
+                Some(code.s("arch_name", 2).pos()),
+                Position::new(1, 34).range_to(Position::new(2, 0)),
```

Comment:
I have really tried to avoid hard-coding positions in the test cases as it makes them very tedious to update. I have made a lot of utilities to find the position of substrings within the code that I always use.